statesync/messages.go

```diff
@@ -1,11 +1,7 @@
 package statesync
```

Comment:
This file doesn't really contain anything anymore. Maybe just move the remains into `reactor.go`?